app/views/mods_panel.py

```diff
@@ -87,9 +98,24 @@ class ModsPanelSortKey(Enum):
     """
     Enum class representing different sorting keys for mods.
     """
+    NOKEY = 0
+    MODNAME = 1
+
 
-    NOKEY = None
-    MODNAME = uuid_to_mod_name
+def sort_uuids(uuids: List[str], key: ModsPanelSortKey) -> List[str]:
+    """
+    Sort the list of UUIDs based on the provided key.
+    Args:
+        key (ModsPanelSortKey): The key to sort the list by.
+    Returns:
+        None
+    """
+    # Sort the list of UUIDs based on the provided key
+    if key == ModsPanelSortKey.MODNAME:
+        key_function = uuid_to_mod_name
+    else:
+        key_function = uuid_no_key
```

Comment:
I think this should just be a lambda function, given how its just returning the uuid? Though it could be better to be more explicit for continuity reasons.

Reply:
Will leave this up to your discretion. Just a suggestion.

Reply:
It was a lambda function, and I changed it to named function because of the pep 8 violation. I don't know how to go through that linting check using a lambda function, do you know what is the correct way of doing it in this case? Thanks a lot!

Reply:
Currently vs code / ai helper has told me to use a comment to skip that checking on that line, I'm not sure it's a good way to do this. What do you think?

Reply:
You should be able to instead do something like


```suggestion
        return sorted(uuids, key=lambda x: x)
```

Its not as clean in that you're repeating the uuids part of that func call. But, it gets around the pep error because you're not assigning the lambda function to a variable. 
As another alternative, use partial from functools to do something like

```suggestion
    pf = partial(sorted, uuids)
    if key == ModsPanelSortKey.MODNAME:
        return pf(key=uuid_to_mod_name)
    else:
        return pf(key=uuid_no_key)
```